index.js

```diff
@@ -19,12 +24,36 @@ let updateScores = () => {
     happyScoreIntoDom.textContent = defaultHappyScore;
     conductScoreIntoDom.textContent = defaultConductScore;
 };
+
+var actionLog = document.querySelector('.actionLog');
+
```

Comment:
actionLog should probably be at the top of the page with the rest of the query selectors.